packages/client/modules/userDashboard/components/GcalModal/VideoConferencing.tsx

```diff
@@ -0,0 +1,65 @@
+import styled from '@emotion/styled'
+import ArrowDropDownIcon from '@mui/icons-material/ArrowDropDown'
+import {Close} from '@mui/icons-material'
+import React from 'react'
+import {MenuPosition} from '../../../../hooks/useCoords'
+import useMenu from '../../../../hooks/useMenu'
+import VideoConferencingMenu from './VideoConferencingMenu'
+import RaisedButton from '../../../../components/RaisedButton'
+import {PALETTE} from '../../../../styles/paletteV3'
+import {Elevation} from '../../../../styles/elevation'
+import GoogleMeetProviderLogo from '../../../../components/GoogleMeetProviderLogo'
+import ZoomProviderLogo from '../../../../components/ZoomProviderLogo'
+
+const StyledButton = styled(RaisedButton)({
+  borderRadius: 4,
+  padding: '6px 12px'
+})
+
+const CloseIcon = styled(Close)({
+  color: PALETTE.SLATE_600,
+  cursor: 'pointer',
+  '&:hover': {
+    opacity: 0.5
+  }
+})
+
+type Props = {
+  videoType: 'meet' | 'zoom' | null
+  handleChangeVideoType: (videoType: 'meet' | 'zoom' | null) => void
+}
+
+const VideoConferencing = (props: Props) => {
+  const {videoType, handleChangeVideoType} = props
+  const {togglePortal, originRef, menuPortal, menuProps} = useMenu(MenuPosition.UPPER_CENTER)
```

Comment:
-1 could you use `@radix-ui/react-dropdown-menu` or another radix component here? They offer SO MUCH bug free great little accessibility features that this old one doesn't. Soon I wanna replace all my crummy menu/modal hacks with radix!

Reply:
I started using Radix but refactored back to the old pattern because it wasn't working well with MUI - see: https://github.com/ParabolInc/parabol/issues/8711. I'm running into a similar issue here with Radix where the parent closes when the menu closes, and the menu items aren't clickable. 

Once https://github.com/ParabolInc/parabol/issues/8711 is resolved, I'd like to refactor the gcal modal away from the old pattern to Radix.

Reply:
Hey @mattkrick, just following up to see if this is good to merge?

Reply:
Hey @mattkrick, bumping this

Reply:
merging now, it'll make the ship this week!